<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startInterpereter">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>142</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Запустить интерпритатор</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="debugLog">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>311</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="programText">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>39</y>
      <width>321</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathToFile">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>321</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>449</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>663</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
